wolfcrypt/src/sp_int.c

```diff
@@ -93,6 +93,12 @@ This library provides single precision (SP) integer math functions.
  * WOLFSSL_SP_FAST_MODEXP       Allow fast mod_exp with small C code
  */
 
+/* WOLFSSL_SP_SMALL is incompatibile with clang-12+ -Os. */
```

Comment:
Spell `incompatible`. Also can we flag this with a `TODO:`. I'm worried the issues here will be overlooked later.